packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -952,6 +955,7 @@ namespace KB = KokkosBatched::Experimental;
         for (local_ordinal_type i=0,iend=pids.send.extent(0);i<iend;++i) {
           copy<ToBuffer>(lids.send, buffer.send, offset_host.send(i), offset_host.send(i+1),
                          mv, blocksize);
+          // FENCE REVIEW: I moved this fence here and I think will be permanent for isend
```

Comment:
@kyungjoo-kim And this one

Reply:
The stream version pack/unpack is not activated in the code. So, fence or no fence does not change the code behavior. I am actually working on this part and will handle this myself.